byway/src/main/webapp/script.js

```diff
@@ -15,14 +15,144 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+window.onload = function() {
+  setProgressBar(1);
+  getStartDestination();
+  getLocations();
+};
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function setProgressBar(pageNumber){
+  var ul = document.getElementById("progressbar");
```

Comment:
Lets switch to using `let` or `const` instead of `var` in our javascript files. It is generally considered a good practice as `var` is globally scoped and can lead to all kinds of weird errors and glitches. You can read more about the differences here:
https://www.javascripttutorial.net/es6/difference-between-var-and-let/

Reply:
done